bsp/src/mill/bsp/MillBuildServer.scala

```diff
@@ -625,6 +627,12 @@ class MillBuildServer(
       new CleanCacheResult(msg, cleaned)
     }
 
+  override def debugSessionStart(debugParams: DebugSessionParams)
+      : CompletableFuture[DebugSessionAddress] =
+    completable(s"debugSessionStart ${debugParams}") { state =>
+      throw new NotImplementedError("debugSessionStart endpoint is not implemented")
+    }
```

Comment:
There's a `DebugProvider` capability that is not set by the user so it's safe to throw an error here from the BSP client perspective.